src/js/tabs/osd.js

```diff
@@ -1369,6 +1374,87 @@ OSD.loadDisplayFields = function() {
             positionable: true,
             preview: 'AUX',
         },
+        SYS_GOGGLE_VOLTAGE: {
+            name: 'SYS_GOGGLE_VOLTAGE',
+            text: 'osdTextElementSysGoggleVoltage',
+            desc: 'osdDescElementSysGoggleVoltage',
+            defaultPosition: -1,
+            draw_order: 485,
+            positionable: true,
+            preview: 'G 16.8V',
+        },
+        SYS_VTX_VOLTAGE: {
+            name: 'SYS_VTX_VOLTAGE',
+            text: 'osdTextElementSysVtxVoltage',
+            desc: 'osdDescElementSysVtxVoltage',
+            defaultPosition: -1,
+            draw_order: 490,
+            positionable: true,
+            preview: 'A 12.6V',
+        },
+        SYS_BITRATE: {
+            name: 'SYS_BITRATE',
+            text: 'osdTextElementSysBitrate',
+            desc: 'osdDescElementSysBitrate',
+            defaultPosition: -1,
+            draw_order: 495,
+            positionable: true,
+            preview: '50MBPS',
+        },
+        SYS_DELAY: {
+            name: 'SYS_DELAY',
+            text: 'osdTextElementSysDelay',
+            desc: 'osdDescElementSysDelay',
+            defaultPosition: -1,
+            draw_order: 500,
+            positionable: true,
+            preview: '24.5MS',
+        },
+        SYS_DISTANCE: {
+            name: 'SYS_DISTANCE',
+            text: 'osdTextElementSysDistance',
+            desc: 'osdDescElementSysDistance',
+            defaultPosition: -1,
+            draw_order: 505,
+            positionable: true,
+            preview: '10${FONT.symbol(SYM.METRE)}',
+        },
+        SYS_LQ: {
+            name: 'SYS_LQ',
+            text: 'osdTextElementSysLQ',
+            desc: 'osdDescElementSysLQ',
+            defaultPosition: -1,
+            draw_order: 510,
+            positionable: true,
+            preview: 'G${FONT.symbol(SYM.LINK_QUALITY)}100',
+        },
+        SYS_GOGGLE_DVR: {
+            name: 'SYS_GOGGLE_DVR',
+            text: 'osdTextElementSysGoggleDVR',
+            desc: 'osdDescElementSysGoggleDVR',
+            defaultPosition: -1,
+            draw_order: 515,
+            positionable: true,
+            preview: '${FONT.symbol(SYM.ARROW_SMALL_RIGHT)}G DVR 8.4G',
+        },
+        SYS_VTX_DVR: {
+            name: 'SYS_VTX_DVR',
+            text: 'osdTextElementSysVtxDVR',
+            desc: 'osdDescElementSysVtxDVR',
+            defaultPosition: -1,
+            draw_order: 520,
+            positionable: true,
+            preview: '${FONT.symbol(SYM.ARROW_SMALL_RIGHT)}A DVR 1.6G',
```

Comment:
```suggestion
            preview: `${FONT.symbol(SYM.ARROW_SMALL_RIGHT)}A DVR 1.6G`,
```

Reply:
But they are... 
![image](https://user-images.githubusercontent.com/11480839/201537623-48ca0954-55f5-4dbc-8078-ea6dc02c5b86.png)

Happy to change anyway for consistency.

Reply:
Fixed